docs/manual/workflows.rst

```diff
@@ -0,0 +1,98 @@
+.. _manual_workflows:
+
+===============
+Beheerprocessen
+===============
+
+Formulieren worden gemaakt, gepubliceerd, bijgewerkt, vervangen en eventueel
+uitgefaseerd. Hieronder staan de typische beheerprocessen voor formulieren die
+u kunt gebruiken om de levensloop van een formulier zo goed mogelijk te laten
+verlopen.
+
+Essentieel bij het beheer zijn 2 opties:
+
+* **Actief** geeft aan dat een formulier beschikbaar is voor iedereen.
+* **Onderhoudsmodus** geeft aan dat een formulier tijdelijk niet beschikbaar 
+  is.
+
+Zodra een formulier *niet actief* of in *onderhoudsmodus* staat, dan is het
+formulier voor reguliere gebruikers direct niet meer te gebruiken en lopende 
+sessies kunnen niet worden afgemaakt. In het geval een formulier *niet actief*
+is, dan geldt dit ook voor beheerders. In *onderhoudsmodus* kunnen beheerders
+nog wel het formulier starten, invullen en verzenden.
+
+
+Nieuw formulier maken
+=====================
+
+Als u het formulier direct op de productieomgeving bouwt, wilt u niet dat 
+reguliere gebruikers dit formulier al kunnen benaderen of invullen.
+
+Zet de optie **Onderhoudsmodus** aan voordat u de eerste maal op **Opslaan** 
+klikt. Het formulier is hierdoor enkel door te starten door beheerders. 
```

Comment:
There's a nuance at the moment were only form designers can do this (so `user.is_staff` and `user.has_perm("forms.can_change_form")`).

I think we should either document this, or change the SDK logic to also allow `user.is_staff = True` to use those. For that we need to emit more meta-information through response headers, getting to the point that it will be better suited to set up a single header with multiple flags like `X-User-Flags: isStaff=true, isFormDesigner=false`

Reply:
I think I want to skip this nuance from the manual for now, but good catch.